Refit/ProblemDetails.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Refit
+{
+    public class ProblemDetails
+    {
+        public Dictionary<string, string[]> Errors { get; set; } = new Dictionary<string, string[]>();
```

Comment:
@paulcbetts any thoughts on using the concrete type here vs an `IDictionary<string,string>` or `IReadOnlyDictionary<string, string>`?